doc/driver.md

```diff
@@ -355,7 +355,154 @@ let generate_all odocl_files =
 The following code actually executes all of the above, and we're done!
 
 ```ocaml env=e1
-let compiled = compile_all ();;
-let linked = link_all compiled;;
-let _ = generate_all linked;;
+let compiled = compile_all () in
+let linked = link_all compiled in
+generate_all linked
 ```
+
```

Comment:
I think this section ought to be moved out of the driver.md file and placed elsewhere - we could certainly link to it from the driver though!